walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,63 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    
   }
+    
+  private static List<TimeRange> sortTimesByStart(List<TimeRange> timeRanges) {
+    timeRanges.sort(TimeRange.ORDER_BY_START);
+    return timeRanges;
+  }
+
+  // TODO(biancamacias): finish implementing ---->
+  private static List<Event> sortEventsByStart(Collection<Event> events) {
+    List<Event> sortedEvents = new ArrayList<>();
+    return sortedEvents;
+  }
+
+  // TODO(biancamacias): finish implementing ---->
+  private static List<TimeRange> findClosedTimeRanges(Collection<Event> events, Collection<String> attendees) {
+    List<TimeRange> closedTimeRanges = new ArrayList<>();
+    return closedTimeRanges;
+  }
+
+  private static List<TimeRange> findOpenTimeRanges(Collection<Event> events, Collection<String> attendees, long duration) {
+    List<TimeRange> openTimeRanges = new ArrayList<>();
+    int start = TimeRange.START_OF_DAY;
+    int end = TimeRange.END_OF_DAY;
+    int allday = TimeRange.WHOLE_DAY;
+    int minutesStart = 1440;
```

Comment:
hmm where did the 1440 come from? I might have missed it somewhere but just curious.

Reply:
its the amount of minutes that are in 1 day (60 * 24)